src/server/jobs/prepare-bounties.ts

```diff
@@ -46,19 +46,56 @@ const prepareBounties = createJob('prepare-bounties', '0 1 * * *', async () => {
     >`SELECT
           be.id,
           be."userId",
-          COALESCE(SUM(bb."unitAmount"), 0) AS "awardedUnitAmount"
+          COALESCE(SUM(bb."unitAmount"), 0) AS "awardedUnitAmount",
+          bes."reactionCountAllTime" AS "reactionCountAllTime"
       FROM "BountyEntry" be
+      LEFT JOIN "BountyEntryStat" bes on bes."bountyEntryId" = be.id
       LEFT JOIN "BountyBenefactor" bb ON bb."awardedToId" = be.id AND bb.currency = ${currency}::"Currency"
       WHERE be."bountyId" = ${id}
-      GROUP BY be.id, be."userId" 
-      ORDER BY "awardedUnitAmount" DESC, be.id ASC LIMIT 1
+      GROUP BY be.id, be."userId", bes."reactionCountAllTime" 
+      ORDER BY "awardedUnitAmount" DESC, "reactionCountAllTime" DESC, be.id ASC LIMIT 1
     `;
 
     if (!winnerEntry) {
+      console.log('no winners :( returning monys');
```

Comment:
Maybe use the logger here if this is needed. Also, typo
```suggestion
      log('no winners :( returning money');
```